src/main/java/org/wso2/carbon/connector/Exists.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.connector;
+
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.SynapseException;
+import org.wso2.carbon.connector.core.AbstractConnector;
+import org.wso2.carbon.connector.core.ConnectException;
+import redis.clients.jedis.Jedis;
+
+public class Exists extends AbstractConnector {
+
+    @Override
+    public void connect(MessageContext messageContext) throws ConnectException {
+        try {
+            Jedis jedis;
+            RedisServer serverObj = new RedisServer();
+            jedis = serverObj.connect(messageContext);
+            if (jedis != null) {
+                String key = messageContext.getProperty(RedisConstants.KEY).toString();
+                Boolean response = jedis.exists(key);
```

Comment:
Shouldn't we check response == true here?

Reply:
No need to check because response may be false or true, depends on the request. Then one of both, true and false can  be the response. So here we don't need to check. We can check it in the integration test.

Reply:
Then what if the response is null?

Reply:
Yes we can handle exception if response is null